test/fashion-mnist/infer.py

```diff
@@ -0,0 +1,115 @@
+import torch
+import torch.nn as nn
+import torch.optim as optim
+import torchvision
+import torchvision.transforms as transforms
+
+# Define the neural network model
+class Net(nn.Module):
+    def __init__(self):
+        super(Net, self).__init__()
+        self.fc1 = nn.Linear(784, 256)
+        self.fc2 = nn.Linear(256, 128)
+        self.fc3 = nn.Linear(128, 10)
+
+    def forward(self, x):
+        x = x.view(x.size(0), -1)
+        x = torch.relu(self.fc1(x))
+        x = torch.relu(self.fc2(x))
+        x = self.fc3(x)
+        return x
+
+## IMPROVE
+from improve import framework as frm
+import candle
+from pathlib import Path
+
+filepath = Path(__file__).resolve().parent
+
+
+# Part 3: Model Testing
+## IMPROVE
+def run(params):
+##
+
+   # Need to get testloader from Part 1.
+   transform = transforms.Compose([
+      transforms.ToTensor(),
+      transforms.Normalize((0.5,), (0.5,))
+   ])
+   
+   # Get the data directory, batch size and other hyperparameters from params 
+   ##IMPROVE 
+   batch_size = params["batch_size"]
+   learning_rate = params["learning_rate"]
+   momentum = params["momentum"]
+   dataset_dir = params["data_dir"]
+   
+   # NOTE: using false now for data loading
+   testset = torchvision.datasets.FashionMNIST(root=dataset_dir, train=False, download=False, transform=transform)
+   testloader = torch.utils.data.DataLoader(testset, batch_size=batch_size, shuffle=True)
```

Comment:
Loading test data for inference? I would expect to load the model weights.

Reply:
See line 72, how are you going to get images to test? 
`outputs = model(images)`

Reply:
I don't understand, or is this a problem of naming conventions? We are doing label prediction in this script no testing. Do you have a specific use case in mind?

Reply:
@rajeeja Any thoughts?